source/module_hamilt_general/module_xc/xc_functional_gradcorr.cpp

```diff
@@ -71,6 +73,7 @@ void XC_Functional::gradcorr(double &etxc, double &vtxc, ModuleBase::matrix &v,
 	// calculate the gradient of (rho_core+rho) in reciprocal space.
 	rhotmp1 = new double[rhopw->nrxx];
 	rhogsum1 = new std::complex<double>[rhopw->npw];
+	ModuleBase::Memory::record("XC_Functional::rho*",sizeof(std::complex<double>)*rhopw->npw+sizeof(double)*rhopw->nrxx);
```

Comment:
Please calculate the total memory used in this function, and record it once with key word "XC_Functional::gradcorr".